pkg/logs/config/integration_config.go

```diff
@@ -64,28 +63,26 @@ type IntegrationConfigLogSource struct {
 	Image string // Docker
 	Label string // Docker
 
-	Service         string
-	Logset          string
-	Source          string
-	SourceCategory  string
-	Tags            string
+	Service        string
+	Logset         string
+	Source         string
+	SourceCategory string
+	Tags           string
+	// TODO: should be moved out
```

Comment:
same here?

Reply:
```
	TagsPayload     []byte
	ProcessingRules []LogsProcessingRule `mapstructure:"log_processing_rules"`
```
Same reasons: not part of the configuration, and built at run-time.